dotCMS/src/integration-test/java/com/dotcms/datagen/SiteDataGen.java

```diff
@@ -14,22 +14,38 @@
     private final long currentTime = System.currentTimeMillis();
 
     private String name = "test" + currentTime + ".dotcms.com";
+    private String aliases;
```

Comment:
Codacy found an issue: [Field aliases has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5809706)